lib/runtime/wasmer/imports.go

```diff
@@ -1541,8 +1543,15 @@ func ext_offchain_submit_transaction_version_1(context unsafe.Pointer, data C.in
 //export ext_offchain_timestamp_version_1
 func ext_offchain_timestamp_version_1(context unsafe.Pointer) C.int64_t {
 	logger.Trace("[ext_offchain_timestamp_version_1] executing...")
-	logger.Warn("[ext_offchain_timestamp_version_1] unimplemented")
-	return 0
+	now := time.Now().Unix()
+
+	return C.int64_t(now)
+}
+
+//export ext_offchain_sleep_until_version_1
+func ext_offchain_sleep_until_version_1(context unsafe.Pointer, deadline C.int64_t) {
+	logger.Trace("[ext_offchain_sleep_until_version_1] executing...")
+	time.Sleep(time.Duration(deadline))
 }
```

Comment:
I don't think this is correct, `deadline` is a unix timestamp that the program should sleep until. instead, this is treating a unix time as a duration and sleeping for that long. I think `context.WithDeadline(context.Background(), deadline)` would work. as well, please double check the units that are being passed in (are they unix seconds or nanoseconds?)

Reply:
Thanks for catching the issue! I will commit ta fixed version shortly!